src/Generator/ParameterGenerator.php

```diff
@@ -163,14 +163,20 @@ public function __construct(
     }
 
     /**
-     * @param  string $type
+     * @param  mixed $type
```

Comment:
Instead of mixed, you could say `array|string`.  That way we know exactly what the function is designed to consume